src/lib/ddl/shopping-ddl.js

```diff
@@ -0,0 +1,42 @@
+/* @flow */
+import { createEventHandler } from './event-handlers';
+
+const snippetRunEvent = { event: 'snippetRun', t: new Date().getTime() };
+export const DDL_NAME = 'shoppingDDL';
+
+const _getDDL = (w) => {
+  let ddl = w[DDL_NAME];
+
+  if (!ddl) {
+    ddl = [ snippetRunEvent ];
+  }
+
+  return ddl;
+};
+
+export const setupShoppingDDL = (w : any, shoppingAnalytics : any) => {
+  const unprocessedEvents = [];
```

Comment:
we can stick with a naming convention - either `unprocessed` or `unconsumed` events